source/slow-down.ts

```diff
@@ -94,7 +94,7 @@ export const slowDown = (
 			? { default: notUndefinedOptions.validate }
 			: notUndefinedOptions.validate ?? { default: true }
 
-	// TODO: Remove in v3.
+	// Deprecation warning for delayMs behavior change from v2
```

Comment:
Oof, I should probably actually remove this check. I think it was important to have there when we first released v2, but I don't think it's needed at this point. I just forgot to remove it when we did the v3 bump.

Doesn't need to be part of this PR, though.